aten/src/TH/THGeneral.cpp

```diff
@@ -275,13 +275,19 @@ void THSetNumThreads(int num_threads)
   // See https://github.com/pytorch/pytorch/issues/13757
   mkl_set_dynamic(false);
 #endif
-
+#if !defined(_OPENMP) && !defined(TH_BLAS_MKL)
+  if (num_threads != 1)
+    THError("PyTorch is compiled with neither OpenMP nor MKL. Setting the "
```

Comment:
I think these sorts of things should be warnings (not errors) particularly because it's hard for users to guard the calls to `torch.set_num_threads` appropriately. (i.e. it's hard for them to determine programmatically if PyTorch was compiled with _OPENMP or TH_BLAS_MKL).

Reply:
Makes sense. Will make it a warning. :)